proposals/2020-11-inflation-rate-change/README.md

```diff
@@ -0,0 +1,39 @@
+﻿
+Cosmos Hub Inflation variable modification: Inflation Rate Change
+
+**Quick Summary of issue**
+
+There are 6 main variables that control the maximum, minimum, & change of the atom inflation rate for the cosmos hub. These variables can be found inside the mint module via the attached hyperlink: [https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md](https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md)
+
+In this proposal we will be looking at adjusting the inflation rate change variable.
+
+- When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year if a shock unbond occurred. Thusly, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.
+
+**How to select “inflation rate change” AKA Maximum Slope of the inflation curve**
+
+The purpose of changing the networks inflation rate is to protect it from unbonding shocks that can sacrifice the security of the network. When the bonded ratio gets below the goal bonded ratio (66% on the hub currently) the inflation rate & effective yield of staked atoms goes up to try and incentive new atoms to bond and secure the network. Inversely, if we regain a proper amount of staked tokens, the yield will drop and thusly decrease effective yields for all delegators. Ideally the inflation rate starts changing fast, optimizing network security over monetary hardness. To figure out what an appropriate selection for the cosmos hub would be, I created excel sheets to run through all the different scenarios to find what value made the inflation rate react in an optimal manner. **I came to the conclusion that 1 (AKA 100% per year is the maximum slope of the inflation curve) gave the most ideal characteristics, with the added benefit of simplifying the equation**. I’ll give some examples on how it would react. Since unbonding is what we are protecting against, I will look at flash unbonds while the inflation rate is at the floor due to that being the #1 time of vulnerability.

```

Comment:
```suggestion
The purpose of changing the network's inflation rate is to protect it from unbonding shocks that can threaten to compromise the security of the network. When the bonded ratio gets below the goal bonded ratio (currently at 66% on the hub) the inflation rate & effective yield of staked atoms goes up in order to incentivize holders to bond new ATOMs with a view to securing the network. Inversely, if we regain a desired amount of staked tokens, the yield will drop and thusly decrease effective yields for all delegators. Ideally the inflation rate starts changing fast, optimizing network security over monetary hardness. To figure out what an appropriate selection for the cosmos hub would be, I created excel sheets to run through all the different scenarios to find what value made the inflation rate react in an optimal manner. **I came to the conclusion that 1 (AKA 100% per year is the maximum slope of the inflation curve) gave the most ideal characteristics, with the added benefit of simplifying the equation**. I’ll give some examples on how it would react. Since unbonding is what we are protecting against, I will look at flash unbonds while the inflation rate is at the floor due to that being the #1 time of vulnerability.

```